packages/cosmic-swingset/src/agcc-shim.js

```
@@ -0,0 +1,21 @@
+/* global process */
+import { Fail } from '@agoric/assert';
+
+let agccP;
+const { AGVM_FROM_AGD, AGVM_TO_AGD } = process.env;
+if (AGVM_FROM_AGD && AGVM_TO_AGD) {
+  const fd = [parseInt(AGVM_FROM_AGD, 10), parseInt(AGVM_TO_AGD, 10)];
```

`parseInt` isn't great for CLI argument validation because it silently truncates the input (e.g., "3.14" becomes `3`, "4x" becomes `4`, and "0x7E" becomes `0`). So instead, I'm advocating for `const parseNumber = str => (/[0-9]/.test(str || '') ? Number(str) : NaN)`.